### DISCUSSION TITLE
Bind the next code feature to a setting

### DISCUSSION THREAD

**Discussion by Ardakilic** [ente-io/ente#1025] [Enhancements]
Hello, first of all thanks for the app, it's much appreciated!

I believe It would be great if there would be an option to toggle the shown next code.

Assuming my device is eavesdropped or compromised somehow, even for a second, this ultimately doubles the period to unlock the second factor, because my device shows the next token that would be generated. So I believe having this feature enabled or disabled via a setting would be great.

I hope this is considered.

Thanks again, and have a great day!

**Comment by ados8** [comment]
I would really like to have an option to change the current and next code to current and past code.
Right now the highlighted/focused code is the current.
There are soo many times where your nearly done entering the code and it's gone.
Knowing the next code is ok but it's not as visible.
